build.proj

```diff
@@ -18,8 +18,9 @@
 
   <Target Name="BulkRestoreNugetPackages"
           BeforeTargets="Build;BuildAllProjects"
-          DependsOnTargets="_RestoreBuildTools">
+          DependsOnTargets="_RestoreBuildTools"
+          Condition="'$(SolutionFile)'!='' and Exists('$(SolutionFile)')">
```

Comment:
If the SolutionFile is not set, nuget.exe won't process the solution file. In that case the targets will process it using dnu.